python/ml4ir/base/pipeline.py

```diff
@@ -579,6 +579,8 @@ def run_pipeline(self, relevance_dataset=None):
                     group_metrics_min_queries=self.args.group_metrics_min_queries,
                     logs_dir=self.logs_dir_local,
                     compute_intermediate_stats=self.args.compute_intermediate_stats,
+                    evaluation_config_path=self.args.evaluation_config,
```

Comment:
@mohazahran I recommend reading the yaml file and passing a dictionary here instead.